components/css/global/_base.scss

```diff
@@ -10,77 +10,66 @@ body{
 	body{

```

Comment:
Throughout this file, I modified the font sizes to use our function and consolidated redundancies.